target/linux/lantiq/files/arch/mips/boot/dts/lantiq/ar10.dtsi

```diff
@@ -0,0 +1,458 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/phy/phy-lantiq-vrx200-pcie.h>
+
+/ {
+ 	#address-cells = <1>;
+ 	#size-cells = <1>;
+ 	compatible = "lantiq,xway", "lantiq,ar10";
+
+ 	aliases {
+ 		serial0 = &asc1;
+ 	};
+
+ 	chosen {
+ 		stdout-path = "serial0:115200n8";
+ 	};
+
+ 	cpus {
+ 		cpu@0 {
+ 			compatible = "mips,mips34Kc";
+ 		};
+ 		cpu@1 {
+ 			compatible = "mips,mips34Kc";
+ 		};
+ 	};
+
+ 	memory@0 {
+ 		device_type = "memory";
+ 	};
+
+ 	cputemp@0 {
+ 		compatible = "lantiq,cputemp";
+ 	};
+
+ 	biu@1f800000 {
+ 		#address-cells = <1>;
+ 		#size-cells = <1>;
+ 		compatible = "lantiq,biu", "simple-bus";
+ 		reg = <0x1f800000 0x800000>;
+ 		ranges = <0x0 0x1f800000 0x7fffff>;
+
+ 		icu0: icu0@80200 {
+ 			#interrupt-cells = <1>;
+ 			interrupt-controller;
+ 			compatible = "lantiq,icu";
+ 			reg = <0x80200 0x28
+ 				0x80228 0x28
+ 				0x80250 0x28
+ 				0x80278 0x28
+ 				0x802a0 0x28>;
+ 		};
+
+ 		icu1: icu1@80300 {
+ 			#interrupt-cells = <1>;
+ 			interrupt-controller;
+ 			compatible = "lantiq,icu1";
+ 			reg = <0x80300 0x28
+ 				0x80328 0x28
+ 				0x80350 0x28
+ 				0x80378 0x28
+ 				0x803a0 0x28>;
+ 		};
+
+ 		watchdog@803f0 {
+ 			compatible = "lantiq,xrx100-wdt";
+ 			reg = <0x803f0 0x10>;
+ 			regmap = <&rcu0>;
+ 		};
+ 	};
+
+ 	sram@1f000000 {
+ 		#address-cells = <1>;
+ 		#size-cells = <1>;
+ 		compatible = "lantiq,sram", "simple-bus";
+ 		reg = <0x1f000000 0x800000>;
+ 		ranges = <0x0 0x1f000000 0x7fffff>;
+
+ 		eiu0: eiu@101000 {
+ 			#interrupt-cells = <1>;
+ 			interrupt-controller;
+ 			compatible = "lantiq,eiu-xway";
+ 			reg = <0x101000 0x1000>;
+ 			interrupt-parent = <&icu0>;
+ 			lantiq,eiu-irqs = <166 135 66 40 41 42>;
+ 		};
+
+ 		pmu0: pmu@102000 {
+ 			compatible = "lantiq,pmu-xway";
+ 			reg = <0x102000 0x1000>;
+ 		};
+
+ 		cgu0: cgu@103000 {
+ 			compatible = "lantiq,cgu-xway";
+ 			reg = <0x103000 0x1000>;
+ 		};
+
+		ts: ts@106F00 {
+			compatible = "lantiq,ts-xrx300";
+			reg = <0x106F00 0x10>;
+			interrupt-parent = <&icu0>;
+			interrupts = <27>;
+			lantiq,numofsensors = <0x1>;
+		};
+
+
+ 		dcdc@106a00 {
+ 			compatible = "lantiq,dcdc-xrx200";
+ 			reg = <0x106a00 0x200>;
+ 		};
+
+ 		vmmc: vmmc@103000 {
```

Comment:
VMMC (Voice Micro Microcontroller??) is used only for VoiP. This device doesnt have FXS and this node is unnececary.